spec/wrappers/prospect_api_wrapper_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe ProspectApiWrapper do
+  describe '.get_prospects' do
+    it 'returns prospect attributes from API' do
+      stub_const 'ProspectApiWrapper::API_KEY', 'abc'
+      allow(RestClient).to receive(:get)
```

Comment:
this is how Rubymine auto-formated it for me. Can you please spend a minute tomorrow and show me how would you like to format this file?